backends/magma/ceed-magma.c

```diff
@@ -585,15 +585,19 @@ static int CeedTensorContract_Magma(Ceed ceed,
   return 0;
 }
 
-static int CeedBasisApply_Magma(CeedBasis basis, CeedTransposeMode tmode,
```

Comment:
Try to avoid introducing trailing whitespace. See git config `core.whitespace` for ways to help notice it.